corehq/apps/hqwebapp/static/registry/less/registry.less

```diff
@@ -0,0 +1,19 @@
+@b3-import-variables: '../../../../../../../../node_modules/bootstrap/less/variables';
+@b3-import-mixins: '../../../../../../../../node_modules/bootstrap/less/mixins';
+
+@import '../../hqwebapp/less/_hq/includes/variables.less';
+
+.label-plain {
```

Comment:
Curious if you tried out bootstrap's `label-default`, which is pretty similar, although the text is smaller.

Reply:
I did try that - it has a very dark background:

![image](https://user-images.githubusercontent.com/249606/126461727-1deece2c-7dea-45ce-ae4a-3d597d43dcf2.png)
![image](https://user-images.githubusercontent.com/249606/126461521-aec9c4ae-b350-4e52-bd97-8322420eea95.png)

Reply:
hrm, yeah, that's uglier, and the spacing is weird. The thing that's bugging me a little about the new style is that it looks like a button, with the colored background + border, I think people are going to click on it. What if it just didn't have the background?